receiver/wavefrontreceiver/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
The license will need to be updated before merging, please rebase and apply `make addlicense`